specification/security/resource-manager/Microsoft.Security/preview/2017-08-01-preview/iotSecuritySolutions.json

```diff
@@ -481,86 +483,106 @@
           "type": "string",
           "description": "The recommendation type.",
           "enum": [
-            "OpenPortsOnDevice",
-            "PermissiveFirewallPolicy",
-            "PermissiveFirewallRuleInput",
-            "PermissiveFirewallRuleOut",
-            "OperationSystemNotValid",
-            "UnutilizedMessagesFromAgent",
-            "SecurityTwinConfigurationNotOptimal",
-            "SecurityTwinConfigurationConflict",
-            "IdenticalAuthenticationCredentials",
-            "DenyDefaultIpPolicy",
-            "TooLargeIPRange",
-            "EnableDiagnosticsLog"
+            "IoT_ACRAuthentication",
+            "IoT_AgentSendsUnutilizedMessages",
+            "IoT_Baseline",
+            "IoT_EdgeHubMemOptimize",
+            "IoT_EdgeLoggingOptions",
+            "IoT_InconsistentModuleSettings",
+            "IoT_InstallAgent",
+            "IoT_IPFilter_DenyAll",
+            "IoT_IPFilter_PermissiveRule",
+            "IoT_OpenPorts",
+            "IoT_PermissiveFirewallPolicy",
+            "IoT_PermissiveInputFirewallRules",
+            "IoT_PermissiveOutputFirewallRules",
+            "IoT_PrivilegedDockerOptions",
+            "IoT_SharedCredentials",
+            "IoT_VulnerableTLSCipherSuite"
```

Comment:
Changing enum values is classified as a breaking change. Have any SDK clients been published against this API version?

Reply:
@hagba looking at your enum values, I am thinking how this will impact the developer experience in the generated SDK.
So when a developer types RecommendationType. they will be presented with enum value all starting with IOT_
what is the point of starting everything with IOT?
Because the developer will have start typing IOT_ and then either type A or E etc. to choose their preferred enums.
They are already inside a very well defined namespace of your enum type, so any particular reason all your enum values are prefixed with IOT_?